compiler/stable_mir/src/mir/pretty.rs

```diff
@@ -58,16 +58,16 @@ pub fn pretty_statement(statement: &StatementKind) -> String {
             pretty.push_str(format!("        _{} = ", place.local).as_str());
             pretty.push_str(format!("{}", &pretty_rvalue(rval)).as_str());
         }
-        StatementKind::FakeRead(_, _) => todo!(),
```

Comment:
Can we print some sort of TODO message for now instead of silently ignoring the statement?

Reply:
Now these return `StatementKind::X:Unimplemented`